### ISSUE TITLE
Tag 1.0.0

### ISSUE THREAD

**Issue by ekes** [localgovdrupal/localgov_guides#46]
As per localgovdrupal/localgov#194 localgovdrupal/localgov#200

 * Bug #45 for #44 should be merged first; #37 is an enhancement that would be good to have.
 * Is #31 blocking

**Comment by ekes** [comment]
#45 #44 - merged and fixed.
#37 Nice to have, requires tests running, to do so could make a new PR
#31 is technically fixed but architecturally might want more work, can come later. Merging #49 at least.

**Comment by ekes** [comment]
https://github.com/localgovdrupal/localgov_guides/releases/tag/1.0.0
